keycloak/keycloak-admin-ui
# Add form for User Profile attribute groups

Implements the Attribute Groups under the User Profile as per [design](https://marvelapp.com/prototype/5eed532/section/1460646). This allows Attribute Groups to be created and edited using a form.

## Verification Steps
1. Enable 'User Profile' under the realm settings and save the settings.
2. Go to 'User Profile' :arrow_right: 'Attributes Groups' and create a group.
3. Verify that you return to the overview and the new entry is displayed.
4. Edit the newly created entry and verify that it is saved correctly.

Tests will follow in a separate PR (see #2022).
